homeassistant/components/steam_online/sensor.py

```diff
@@ -170,6 +172,10 @@ def device_state_attributes(self):
         attr = {}
         if self._game is not None:
             attr["game"] = self._game
+        if self._game_id is not None:
+            attr["game_id"] = self._game_id
+            attr["game_image_header"] = f"https://steamcdn-a.akamaihd.net/steam/apps/{self._game_id}/header.jpg"
+            attr["game_image_main"] = f"https://steamcdn-a.akamaihd.net/steam/apps/{self._game_id}/capsule_616x353.jpg"
```

Comment:
Wouldn't it be useful to utilize [entity_picture](https://developers.home-assistant.io/docs/entity_index/#generic-properties) for one of this pictures?

Please create a constant for `https://steamcdn-a.akamaihd.net/steam/apps/` to reuse it.

Reply:
Good point .. will do thanks :)

Reply:
Done both of those changes you suggested; I used the header as the entity_picture (as it is quite small) when in-game, and the profile picture when not.